client/post-editor/editor-publish-button/test/index.jsx

```diff
@@ -130,6 +130,7 @@ describe( 'EditorPublishButton', () => {
 						savedPost={ { status: 'future', date: lastMonth } }
 						post={ { title: 'change', status: 'future', date: lastMonth } }
 						site={ MOCK_SITE }
```

Comment:
The `EditorPublishButton` component no longer needs the `site` prop: everything it needs is loaded by `connect` from the selected site.